src/components/icon/icon.component.tsx

```diff
@@ -93,6 +96,13 @@ const Icon = React.forwardRef<HTMLSpanElement, IconProps>(
         ICON_TOOLTIP_POSITIONS.includes(override)
       );
 
+    if (!deprecatedExtraSmallBgSizeTriggered && bgSize === "extra-small") {
```

Comment:
**comment**: I still think its worth adding a deprecation warning if `fontSize` has exceeded `bgSize` to let consumers now that the `bgSize` has been adjusted accordingly